neptune/new/run.py

```diff
@@ -157,7 +161,28 @@ def assign(self, value, wait: bool = False) -> None:
         .. _assign docs page:
             https://docs.neptune.ai/api-reference/run#assign
         """
-        self[""].assign(value, wait)
+        self.root.assign(value, wait)
+
+    def fetch(self) -> dict:
+        """Fetch values of multiple fields as a dictionary.
```

Comment:
@Herudaio can you comment/approve on docstring for fetch (https://neptune-labs.atlassian.net/browse/NPT-10005)?